optimizely/client/client_test.go

```diff
@@ -210,7 +209,7 @@ func TestIsFeatureEnabled(t *testing.T) {
 	mockConfig := new(MockProjectConfig)
 	mockConfig.On("GetFeatureByKey", testFeatureKey).Return(testFeature, nil)
 	mockConfigManager := new(MockProjectConfigManager)
-	mockConfigManager.On("GetConfig").Return(mockConfig)
+	mockConfigManager.On("GetConfig").Return(mockConfig, (error)(nil))
```

Comment:
Any reason why you have to cast the nil using `(error) (nil)`? I thought just returning `nil` would work

Reply:
I could have sworn it was giving be an error without it, but not now. Making this change, then merging.